src/components/RecipeView.js

```diff
@@ -3,29 +3,22 @@ import Pagination from "./Pagination";
 import Recipe from "./Recipe";
 import "../styles/RecipeView.css";
 
-class RecipeView extends React.Component {
+export default function RecipeView(props) {
+  const { onPageChange, data, currentPage, total, pageSize } = props;
 
-  displayRecipes = () => {
-    const recipes = this.props.data.map((recipe) => {
-      return <Recipe data={recipe} key={recipe.id}/>
-    })
-    return recipes;
-  }
-  
-  render() {
-    return (
-      <div className="container">
-        {this.displayRecipes()}
-        <Pagination 
-          className="pagination-bar"
-          currentPage={this.props.currentPage}
-          total={this.props.total}
-          pageSize={this.props.pageSize}
-          onPageChange={page => this.props.onPageChange(page)}
+  return (
+    <>
+      {data.map((recipe) => {
+        return <Recipe data={recipe} key={recipe._id} />;
+      })}
+      <div className="pagination-bar">
+        <Pagination
+          currentPage={currentPage}
+          total={total}
```

Comment:
[doesn't need to be changed in this PR] - we should change this to something like totalPages or totalCount to specify what this number actually is

Reply:
acknowledged